escpos/helpers.py

```diff
@@ -44,23 +44,27 @@ def check(self):
 def chunks(iterable, size):
     def chunk_factory(iterable, size):
         args = [iter(iterable)] * size
-        return itertools.izip_longest(*args, fillvalue=None)
+
+        # Norberto Hideaki Enomoto -> Python 3.4
+        #return itertools.izip_longest(*args, fillvalue=None)
+        return itertools.zip_longest(*args, fillvalue=None)
     for chunk in chunk_factory(iterable, size):
         yield ''.join([e for e in chunk if e is not None])
     raise StopIteration()
 
 
 def hexdump(data):
     def _cut(sequence, size):
-        for i in xrange(0, len(sequence), size):
+        for i in range(0, len(sequence), size):
             yield sequence[i:i+size]
     _hex = lambda seq: ['{0:02x}'.format(b) for b in seq]
     _chr = lambda seq: [chr(b) if 32 <= b <= 126 else '.' for b in seq]
     raw_data = map(ord, data)
     hexpanel = [' '.join(line) for line in _cut(_hex(raw_data), 16)]
     chrpanel = [''.join(line) for line in _cut(_chr(raw_data), 16)]
     hexpanel[-1] = hexpanel[-1] + (chr(32) * (47 - len(hexpanel[-1])))
-    chrpanel[-1] = chrpanel[-1] + (chr(32) * (16 - len(chrpanel[-1])))
+    # Norberto Hideaki Enomoto -> error no Django
+    #chrpanel[-1] = chrpanel[-1] + (chr(32) * (16 - len(chrpanel[-1])))
```

Comment:
I do not understand why you comment the padding for the character panel's last line. You marked as "erro on Django"... which error, and how this padding affects Django (in which part)?